e2e/utils/appcmanager.go

```diff
@@ -60,10 +64,20 @@ func (a *AppControllerManager) Run() {
 	}
 }
 
+func (a *AppControllerManager) DeletePod() {
```

Comment:
Are you sure?
a) DeletePod wasn't written by me. In fact, if you take a look at current master, which was reviewed you can find it there
b) TearDown is something that should be executed after each tests. This is a helper method that is executed by one of the tests
c) Maybe it would be a better idea to call it DelteAppControllerPod or to make it have podName as a parameter? `DeletePod` suggests that it just deletes any pod